test/integration/adapters/mailgun_test.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Swoosh.Integration.MailgunAdapterTest do
+  use ExUnit.Case, async: true
+
+  alias Swoosh.Adapters.Mailgun
+  import Swoosh.Email
+
+  @tag external: true
```

Comment:
Is that for the following test case or the whole module? If the latter we should use `@moduletags` I think